torchvision/io/_video_opt.py

```diff
@@ -495,13 +514,17 @@ def get_pts(time_base):
     return vframes, aframes, _info
 
 
-def _read_video_timestamps(filename, pts_unit="pts"):
+def _read_video_timestamps(
+    filename: str,
+    pts_unit: str = "pts"
+) -> Tuple[Union[List[int], List[Fraction]], Optional[float]]:
     if pts_unit == "pts":
         warnings.warn(
             "The pts_unit 'pts' gives wrong results and will be removed in a "
             + "follow-up version. Please use pts_unit 'sec'."
         )
 
+    pts: Union[List[int], List[Fraction]]
```

Comment:
Why this line? Type should be clear from the return type of `_read_video_timestamps_from_file()`.

Reply:
So the redefinition of pts on line 527 doesn't please mypy, so I type it upfront to avoid issues over there